AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -361,67 +376,80 @@ - (BOOL)removeAccount {
   return YES;
 }
 
-- (void)acquireTokenSilentlyWithMSALAccount:(MSALAccount *)account {
+- (void)acquireTokenSilentlyWithMSALAccount:(MSALAccount *)account
+                                 uiFallback:(BOOL)uiFallback
+                keyPathForCompletionHandler:(NSString *)completionHandlerKeyPath {
```

Comment:
Why do you need this instead of just `completionHandler`?

Reply:
To check if it's interrupted